docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
 _Generally available in Grafana Enterprise_
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Correlations
+
+### Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana Open Source, Enterprise and Cloud_
```

Comment:
Removing mention of the Cloud availability here and I'll add these to the Cloud What's new and ping you for review
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```